### PR TITLE
Document some SPL changes in PHP-8.4

### PR DESCRIPTION
**Pull request by Girgias**

### CODE REVIEW

File: reference/spl/splobjectstorage.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
     </oointerface>
 
     <oointerface>
-     <interfacename>Iterator</interfacename>
+     <interfacename>SeekableIterator</interfacename>
```
Comment (cmb69) [inline reference/spl/splobjectstorage.xml:36]: Don't we need a changelog entry for this?

**Review by cmb69 — APPROVED** [review]
Looks good (one possibly issue below). Thank you!

File: reference/spl/splobjectstorage.xml (hunk 1 above)
Comment (Girgias) [reply reference/spl/splobjectstorage.xml:36]: Ah yes we do indeed.

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**